doc/recipe/preprocessor.rst

```diff
@@ -180,22 +180,68 @@ To get an overview on data fixes and how to implement new ones, please go to
 
```

Comment:
Adding an explicit anchor (something like, but better than `.. _fx ancillary cell measures:`) will make it easier in the future to keep the links intact.

Reply:
I don't get this comment.There is an anchor right there which I'm also referring to? `.. _Fx variables as cell measures or ancillary variables:`

Reply:
This relates to the use of `:ref:` later on. Sorry, the terminology is a bit convoluted in this space: You are linking directly to the title of the section using what naked restructured text calls [an implicit hyperlink target](https://docutils.sourceforge.io/docs/user/rst/quickref.html#hyperlink-targets) and I think I saw referred to in a sphinx context as an automatic label.

The problem with this is, when at some point in the future we change the heading, all the links break. Also, you cannot refer to this from a different file in the same document.

If instead we put an extra anchor or label here, we can refer to it from anywhere and the validity of the links is ensured by sphinx.

Reply:
But there IS an anchor:

```rst
...
To get an overview on data fixes and how to implement new ones, please go to
:ref:`fixing_data`.

.. _Fx variables as cell measures or ancillary variables:

Fx variables as cell measures or ancillary variables
====================================================
The following preprocessor may require the use of ``fx_variables`` 
to be able to perform the computations:
...
```

It may be hidden in the diff since it's already present in the current code.

Reply:
You are right, sorry! I did indeed miss the anchor because it didn't show up in the diff and also because it is exactly the same text as the title. I think it would be better if it was different, but no need to change it here.